src/parse_funcs.ts

```diff
@@ -0,0 +1,164 @@
+import axios, { AxiosResponse } from "axios";
+import isGitHubUrl = require("is-github-url");
```

Comment:
I was getting an import error here. Where is that function located?

Reply:
It's a dependency. Node Module should be installed from npm.

Either of 

npm install
or
npm install is-github-url axios

should work

Reply:
Ok